test/functional/apps/visualize/group5/_tsvb_time_series.ts

```diff
@@ -594,7 +595,8 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
         after(async () => await visualBuilder.toggleNewChartsLibraryWithDebug(false));
       });
 
-      describe('index pattern selection mode', () => {
+      describe('index pattern selection mode', function () {
+        this.tags('skipFirefox');
```

Comment:
I think originally intention was to run only `basics` suite on FF. Now we are limited with slowness, I suggest we stick to the minimum until we find a solution.